alpakaConfig.cmake

```diff
@@ -1089,6 +1098,11 @@ IF((ALPAKA_ACC_GPU_CUDA_ENABLE OR ALPAKA_ACC_GPU_HIP_ENABLE) AND ALPAKA_CUDA_COM
     SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${_ALPAKA_COMPILE_OPTIONS_STRING}")
 ENDIF()
 
+IF(NOT MSVC)
+    # dependency for std::future
+    TARGET_LINK_LIBRARIES("alpaka" INTERFACE Threads::Threads)
```

Comment:
just generally: the target name `alpaka` does not need to be placed in quotes `""`